std_msgs/Float64 radius
std_msgs/Float64 angle
std_msgs/Float64[4] covariance
std_msgs/Float64[2] start
std_msgs/Float64[2] end
